coalib/bears/requirements/PythonRequirement.py

```diff
@@ -0,0 +1,45 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    more requirements from ``pip``, without writing the name of the class
+    multiple times.
+    """
+
+    def __init__(self, package, version=""):
+        """
+        Constructs a new PythonRequirement, using the PackageRequirement
+        constructor.
+
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version. Leave empty to
+                        install latest.
+        """
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        Creates a tuple of multiple ``pip`` requirements, consisting of
+        either tuples or strings.
```

Comment:
"consisting of either tuples or string"? It should consist of `PythonRequirements`, right?